.github/workflows/create-release-on-tag.yml

```
@@ -22,14 +22,25 @@ jobs:
           tag_name: ${{ github.ref }}
           release_name: PXF Version ${{ github.ref }}
           body: |
-            ## 6.4.2 (09/19/2022)
+            ## 6.5.0 (12/21/2022)
 
             ### Enhancements:
 
+            - [#872](https://github.com/greenplum-db/pxf/pull/872) Enable automation tests to run against PXF FDW extension
+            - [#876](https://github.com/greenplum-db/pxf/pull/876) Parquet write array
+            - [#877](https://github.com/greenplum-db/pxf/pull/877) Update shell scripts to use a more consistent style
+            - [#879](https://github.com/greenplum-db/pxf/pull/879) Add new SPLIT_BY_FILE option for json profiles
```

Are we matching commit names here? if so this should be
```suggestion
            - [#879](https://github.com/greenplum-db/pxf/pull/879) Refactor JsonRecordReader and fix incorrect parsing of JSON objects for multi-line JSON

---

I explicitly changed from PR name here to commit name as the same PR fixed a bug and introduced a new enhancement, so for this section the enhancement only was relevant